backend/src/main/java/de/neuefische/flooooooooooorian/backend/model/User.java

```diff
@@ -0,0 +1,26 @@
+package de.neuefische.flooooooooooorian.backend.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Document(collection = "users")
+@Builder
+public class User {
+
+    @Id
+    private String id;
+    private String username;
+
+    private List<Location> locations;
+    private List<Picture> pictures;
+    private List<Comment> comments;
```

Comment:
do you need this here? then you have duplicate data in your data. Maybe it makes sense to add user id to locations / comments/ pictures and than filter by username when you need all comments from a specific user